.github/workflows/python-package.yml

```diff
@@ -35,6 +35,8 @@ jobs:
         flake8 climt/ --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 climt/ --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    # - name: Test with pytest
-    #  run: |
-    #    pytest
+    - name: Build climt
+      env:
+             CLIMT_ARCH: Linux
```

Comment:
I think this should have only 4 spaces. The actions are not running at all, which usually means there is a formatting error in this file.

Reply:
No, apparently this is ok. But in any case please change the number of spaces to 2 or 4